lib/mocha/configuration.rb

```diff
@@ -303,6 +306,57 @@ def reinstate_undocumented_behaviour_from_v1_9?
       @options[:reinstate_undocumented_behaviour_from_v1_9]
     end
 
+    # Configure whether to perform strict keyword argument comparision. Only supported in Ruby >= 2.7.
+    #
+    # Without this option, positional hash and keyword arguments are treated the same during comparison, which can lead to false
+    # negatives in Ruby >= 3.0 (see examples below). For more details on keyword arguments in Ruby 3, refer to the relevant
```

Comment:
@wasabigeek I've just been reviewing the documentation and I think this should say "false positives" instead of "false negatives":

> Without this option, positional hash and keyword arguments are treated the same during comparison, which can lead to false positives in Ruby >= 3.0.

Looking at the code example below, the comment says:
* "this passes the test", i.e. it's a "positive".
* "but would result in an ArgumentError in practice", i.e. the "positive" is misleading and so is actually a "false positive".

Does that sound correct to you?

Reply:
My opinion is the opposite actually, at least based on a google search (["A false negative indicates no bug when there is one."](https://testfully.io/blog/false-positive-false-negative/), ["False negatives, on the other hand, are given when the test case passes, but there is in fact a bug present in the system and/or the functionality is not working as it should."](https://www.functionize.com/blog/watching-out-for-false-positives-and-false-negatives-in-software-testing)).

That said, I don't think it's intuitive and this definitely tripped me up when I first looked for the right term. If there's a better alternative I'm all for changing it!

Reply:
Ha! Interesting. I'll have a read and do some more thinking! 👍

Reply:
Addressed in 10996795279980d6844461ea1f42512239cc1567.